src/quo2/components/code/code/style.cljs

```diff
@@ -0,0 +1,84 @@
+(ns quo2.components.code.code.style
+  (:require [quo2.foundations.colors :as colors]))
+
+;; Example themes:
+;; https://github.com/react-syntax-highlighter/react-syntax-highlighter/tree/master/src/styles/hljs
+(defn theme
```

Comment:
`theme` was a map, now it's a function (since it uses `color-theme-colors`) and I used `case` because we don't need to compute all colors to only extract one key (as we'd do if it were a map).